ansible/roles/stack-sunbird/templates/sunbird_nodebb.env

```diff
@@ -1,10 +1,15 @@
 # Initialization variable for mongo
 # Ref: https://community.nodebb.org/topic/4325/set-up-nodebb-via-environment-variables
-database=mongo
+database=redis
+# redis database number
+redis_database={{nodebb_redis_database|default('0')}}
```

Comment:
Default please change to a free database in exisitng redis. We will not set default as 0. 
0 will be used only for prod.